src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -15,6 +15,15 @@
 
 namespace NugetForUnity.Ui
 {
+    public enum NugetWindowTab
```

Comment:
Please don't declare multiple public types / enums in one file. Either move it to a separate file or use a nested private type / enum.

Reply:
I discussed this with @igor84 and he would need it to be public for the upcoming plugin expansion feature. So from these two options, I would probably move it to a separate file. Leaving this conversation unresolved, waiting for your response.

Reply:
You can move it to a separate file.